apps/openmw/mwworld/worldimp.cpp

```diff
@@ -1514,7 +1514,10 @@ namespace MWWorld
         ipos.rot[2] = referenceObject.getRefData().getPosition().rot[2];
 
         MWWorld::Ptr placed = copyObjectToCell(ptr, referenceCell, ipos, ptr.getRefData().getCount(), false);
-        adjustPosition(placed, true); // snap to ground
+
+        if (!placed.getClass().isActor())
```

Comment:
actor have physics so no interest in snap them to the ground

Reply:
I removed it in https://github.com/OpenMW/openmw/pull/2490/commits/2f1fb27d18cb9e9c96df06995adb1b586d78aa56 but it's less fun than letting them fall and squash on the ground